File: UnicoVehicle/UnicoVehicle.DTO/VehiclePrice.cs
using System;
namespace UnicoVehicle.DTO
{
    public class VehiclePrice
    {
        public int VehiclePriceId { get; set; }
        public Miscellaneous.Vehicle Vehicle { get; set; }
        public Miscellaneous.Showroom Showroom { get; set; }
        public double Price { get; set; }
    }
}

File: Flutter App/unicoVehicle/lib/Models/VehiclePrice.dart
import './Miscellaneous/Showroom.dart';
import './Miscellaneous/Vehicle.dart';

class VehiclePrice {
  final int vehiclePriceId;
  final Showroom showroom;
  final Vehicle vehicle;
  final double price;

  VehiclePrice({
    this.price,
    this.showroom,
    this.vehicle,
    this.vehiclePriceId,
  });
}
